common/Jackdmp.cpp

```diff
@@ -43,6 +43,10 @@ Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #include "audio_reserve.h"
 #endif
 
+#if HAVE_SYSTEMD == 1
```

Comment:
This style of macro is inconsistent with the rest of the codebase.
Use `#ifdef HAVE_SYSTEMD` instead

Reply:
I had some issues when using `#ifdef HAVE_SYSTEMD`. WAF will define `HAVE_SYSTEMD == 0` in case of systemd is not available. See `build/config.h` for example:
```c
...
#define HAVE_CELT 0
#define HAVE_OPUS 0
#define HAVE_SAMPLERATE 1
#define HAVE_SNDFILE 1
#define HAVE_READLINE 1
#define HAVE_SYSTEMD_SD_DAEMON_H 1
#define HAVE_SYSTEMD 1
...
```

The issue is reproducible with the following source code:
```c
#define VAR 0

#ifdef VAR
#error "DEFINED"
#else
#error "UNDEF"
#endif
```
It fails with
```bash
$ gcc main.c
main.c:4:2: error: #error "DEFINED"
 #error "DEFINED"
  ^
```

Therefore I think the other `#ifdef` could also end up in issues.

Reply:
I see, you are correct then.
So just use `#if HAVE_SYSTEMD`. no need for the == 1 comparison.